cluster/up.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+
+source ./cluster/gocli.sh
+
+num_nodes=${KUBEVIRT_NUM_NODES:-1}
+re='^-?[0-9]+$'
+if ! [[ $num_nodes =~ $re ]] || [[ $num_nodes -lt 1 ]] ; then
+    num_nodes=1
+fi
+
+image="k8s-1.10.4"
+case "${KUBEVIRT_PROVIDER}" in
+"k8s-1.10.4")
+    image="k8s-1.10.4"
+    ;;
+"os-3.10.0")
+    image="os-3.10.0"
```

Comment:
I would prefer to store `k8s-1.10-4` and `os-3.10.0` in config.sh and then source in the script.  The intent of that file is to give devs an at-a-glance idea of the build / test / deploy configuration, so making the kubevirtci images visible there would be useful.

Reply:
Sure we can do that.